web/sites/default/config/lightning_core.versions.yml

```diff
@@ -205,7 +205,7 @@ panelizer_quickedit: 0.0.0
 panels: 0.0.0
 panels_ipe: 0.0.0
 pantheon_advanced_page_cache: 1.0.0
-paragraphs: 0.0.0
+paragraphs: 1.15.0
```

Comment:
@RichardDavies , this reference to paragraphs in lightning_core_versions.yml seems a bit odd. I would have though this would remove.